docs/api-reference/uiSchema.md

```diff
@@ -218,5 +230,38 @@ const uiSchema = {
   "ui:title": "Your password"
 };
 ```
+
+## submitButtonProps
+
+Sometimes it's convenient to change the behavior of the submit button for the form. This is the purpose of the `ui:submitButtonProps` uiSchema directive:
+
+You can pass any other prop to the submit button if you want, by default, this library will set the following options mentioned below for all submit buttons:
+
+### `disabled` option
+
+You can use this option to disable the submit button
+
+### `className` option
+
+You can use this option to specify a class name for the submit button
+
+### `required` option
+
+You can use this option to remove the submit button completely from the form. Nice option, if the form is just for viewing purposes
```

Comment:
Hi @jacqueswho , I'm not comfortable calling this `required` because `required` has a different meaning otherwise in json schema (usually it just means that the field will be shown, but not required), but here, it basically means that the button will just be removed. Why not call it `hidden` instead?

Also, do you want to have a quick video call so we can review this PR faster and in real time? I hope this will speed up reviews :) https://calendly.com/aramaswamis/meeting?month=2022-01